client/blocks/post-relative-time/test/index.jsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect, assert } from 'chai';
```

Comment:
I would recommend using only one assertion style in a single file, in this case `expect` or `assert`.